### ISSUE TITLE
SELECT true Crashes Server

### ISSUE THREAD

**Issue by apavlo** [cmu-db/noisepage#1328]
# Bug Report
## Summary
We currently do not support SELECT queries without a FROM clause that include a boolean:

**Client**
```SQL
noisepage=# SELECT true;
server closed the connection unexpectedly
        This probably means the server terminated abnormally
        before or while processing the request.
The connection to the server was lost. Attempting reset: Failed.
Time: 1141.247 ms (00:01.141)
```

**Server**
```
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
#1  0x00007ffff78d3859 in __GI_abort () at abort.c:79
#2  0x00007ffff7ca8951 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7cb447c in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7cb44e7 in std::terminate() () from /lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7cb4799 in __cxa_throw () from /lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x0000555557a2d2fd in noisepage::execution::compiler::CompilationContext::Prepare (this=0x7fffeedfae10, expression=...) at /home/wanshenl/BieberFanClub/noisepage/src/execution/compiler/compilation_context.cpp:385
#7  0x00005555579d0b9c in noisepage::execution::compiler::OperatorTranslator::OperatorTranslator (this=0x7fff60008c60, plan=..., compilation_context=0x7fffeedfae10, pipeline=0x7fffeedfab80, 
    feature_type=noisepage::brain::ExecutionOperatingUnitType::OUTPUT) at /home/wanshenl/BieberFanClub/noisepage/src/execution/compiler/operator/operator_translator.cpp:29
#8  0x00005555579d21ae in noisepage::execution::compiler::OutputTranslator::OutputTranslator (this=0x7fff60008c60, plan=..., compilation_context=0x7fffeedfae10, pipeline=0x7fffeedfab80)
    at /home/wanshenl/BieberFanClub/noisepage/src/execution/compiler/operator/output_translator.cpp:21
#9  0x0000555557a3592f in std::make_unique<noisepage::execution::compiler::OutputTranslator, noisepage::planner::AbstractPlanNode const&, noisepage::execution::compiler::CompilationContext*, noisepage::execution::compiler::Pipeline*&> () at /usr/include/c++/9/bits/unique_ptr.h:857
#10 0x0000555557a2bff4 in noisepage::execution::compiler::CompilationContext::PrepareOut (this=0x7fffeedfae10, plan=..., pipeline=0x7fffeedfab80)
    at /home/wanshenl/BieberFanClub/noisepage/src/execution/compiler/compilation_context.cpp:216
#11 0x0000555557a2b4c2 in noisepage::execution::compiler::CompilationContext::GeneratePlan (this=0x7fffeedfae10, plan=...) at /home/wanshenl/BieberFanClub/noisepage/src/execution/compiler/compilation_context.cpp:127
#12 0x0000555557a2be42 in noisepage::execution::compiler::CompilationContext::Compile (plan=..., exec_settings=..., accessor=0x7fff600055b0, mode=noisepage::execution::compiler::CompilationMode::Interleaved, 
    query_text=...) at /home/wanshenl/BieberFanClub/noisepage/src/execution/compiler/compilation_context.cpp:202
#13 0x00005555572685e8 in noisepage::trafficcop::TrafficCop::CodegenPhysicalPlan (this=0x555558e2ddf0, connection_ctx=..., out=..., portal=...)
    at /home/wanshenl/BieberFanClub/noisepage/src/traffic_cop/traffic_cop.cpp:361
#14 0x0000555557652b87 in noisepage::network::ExecutePortal (connection_ctx=..., portal=..., out=..., t_cop=..., explicit_txn_block=false)
    at /home/wanshenl/BieberFanClub/noisepage/src/network/postgres/postgres_network_commands.cpp:35
#15 0x0000555557653afc in noisepage::network::SimpleQueryCommand::Exec (this=0x7fff60005600, interpreter=..., out=..., t_cop=..., connection=...)
    at /home/wanshenl/BieberFanClub/noisepage/src/network/postgres/postgres_network_commands.cpp:181
#16 0x0000555557658abb in noisepage::network::PostgresProtocolInterpreter::Process (this=0x7fff5c004e90, in=..., out=..., t_cop=..., context=...)
    at /home/wanshenl/BieberFanClub/noisepage/src/network/postgres/postgres_protocol_interpreter.cpp:45
#17 0x000055555768dc45 in noisepage::network::ConnectionHandle::Process (this=0x7fff60000ef0) at /home/wanshenl/BieberFanClub/noisepage/src/network/connection_handle.cpp:182
#18 0x000055555769622f in noisepage::network::ConnectionHandleStateMachineTransition::Process (handle=...) at /home/wanshenl/BieberFanClub/noisepage/src/network/connection_handle.cpp:76
#19 0x000055555768d6d1 in noisepage::network::ConnectionHandle::StateMachine::Accept (this=0x7fff60000f10, action=noisepage::network::Transition::WAKEUP, handle=...)
    at /home/wanshenl/BieberFanClub/noisepage/src/network/connection_handle.cpp:129
#20 0x000055555768db20 in noisepage::network::ConnectionHandle::HandleEvent (this=0x7fff60000ef0, fd=125, flags=2) at /home/wanshenl/BieberFanClub/noisepage/src/network/connection_handle.cpp:169
#21 0x000055555768de3c in noisepage::network::ConnectionHandle::<lambda(int, int16_t, void*)>::operator()(int, int16_t, void *) const (__closure=0x0, fd=125, flags=2, arg=0x7fff60000ef0)
    at /home/wanshenl/BieberFanClub/noisepage/src/network/connection_handle.cpp:217
#22 0x000055555768de72 in noisepage::network::ConnectionHandle::<lambda(int, int16_t, void*)>::_FUN(int, int16_t, void *) () at /home/wanshenl/BieberFanClub/noisepage/src/network/connection_handle.cpp:218
#23 0x00007ffff7ece13f in ?? () from /lib/x86_64-linux-gnu/libevent-2.1.so.7
#24 0x00007ffff7ece87f in event_base_loop () from /lib/x86_64-linux-gnu/libevent-2.1.so.7
#25 0x000055555767a177 in noisepage::EventUtil::EventBaseDispatch (base=0x7fff5c000ff0) at ../src/include/common/event_util.h:100
#26 0x000055555767a1b4 in noisepage::common::NotifiableTask::EventLoop (this=0x7fff5c000ee0) at ../src/include/common/notifiable_task.h:151
#27 0x000055555767a1f2 in noisepage::common::NotifiableTask::RunTask (this=0x7fff5c000ee0) at ../src/include/common/notifiable_task.h:158
#28 0x000055555769e44a in noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}::operator()() const (this=0x555558d22330)
    at ../src/include/common/dedicated_thread_registry.h:82
#29 0x00005555576c06db in std::__invoke_impl<void, noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}>(std::__invoke_other, noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}&&) (__f=...) at /usr/include/c++/9/bits/invoke.h:60
#30 0x00005555576c0625 in std::__invoke<noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}>(noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}&&) (__fn=...) at /usr/include/c++/9/bits/invoke.h:95
#31 0x00005555576c0596 in std::thread::_Invoker<std::tuple<noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}> >::_M_invoke<0ul>(std::_Index_tuple<0ul>) (this=0x7fff5c001d58) at /usr/include/c++/9/thread:244
#32 0x00005555576c050e in std::thread::_Invoker<std::tuple<noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}> >::operator()() (
    this=0x7fff5c001d58) at /usr/include/c++/9/thread:251
#33 0x00005555576c03c2 in std::thread::_State_impl<std::thread::_Invoker<std::tuple<noisepage::common::DedicatedThreadRegistry::RegisterDedicatedThread<noisepage::network::ConnectionHandlerTask, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory> >(noisepage::common::DedicatedThreadOwner*, unsigned int, noisepage::common::ManagedPointer<noisepage::network::ConnectionHandleFactory>)::{lambda()#1}> > >::_M_run() (this=0x7fff5c001d50) at /usr/include/c++/9/thread:195
#34 0x00007ffff7ce0d84 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#35 0x00007ffff7e3a609 in start_thread (arg=<optimized out>) at pthread_create.c:477
#36 0x00007ffff79d0293 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```

**Comment by lmwnshn** [comment]
Should be fixed by #1327.

**Comment by mbutrovich** [comment]
Does indeed look fixed after merging #1327.

```SQL
noisepage=# select true;
 ?column? 
----------
 t
(1 row)

Time: 4.957 ms
noisepage=# 
```
